diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Config.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Config.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Config.php	2004-12-05 14:00:30.903001616 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Config.php	2004-12-05 14:01:44.188860472 +0100
@@ -61,7 +61,7 @@
     public function __construct()
     {
         // Load the game config
-        require 'Games.php';
+        require 'Net/GameServerQuery/Games.php';
         $this->_games   = $games;
         $this->_packets = $packets;
         $this->_normals = $normals;
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Breed.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Breed.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Breed.php	2004-12-05 14:00:30.912000248 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Breed.php	2004-12-05 14:02:28.706092824 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Doom3.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Doom3.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Doom3.php	2004-12-05 14:00:30.908000856 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Doom3.php	2004-12-05 14:02:28.709092368 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/FarCry.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/FarCry.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/FarCry.php	2004-12-05 14:00:30.908000856 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/FarCry.php	2004-12-05 14:02:28.713091760 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Freelancer.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Freelancer.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Freelancer.php	2004-12-05 14:00:30.910000552 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Freelancer.php	2004-12-05 14:02:28.714091608 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GameSpy04.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GameSpy04.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GameSpy04.php	2004-12-05 14:00:30.909000704 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GameSpy04.php	2004-12-05 14:02:28.720090696 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GhostRecon.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GhostRecon.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GhostRecon.php	2004-12-05 14:00:30.909000704 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/GhostRecon.php	2004-12-05 14:02:28.722090392 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HalfLife.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HalfLife.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HalfLife.php	2004-12-05 14:00:30.904001464 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HalfLife.php	2004-12-05 14:02:28.727089632 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Hexen2.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Hexen2.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Hexen2.php	2004-12-05 14:00:30.911000400 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Hexen2.php	2004-12-05 14:02:28.729089328 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HomeWorld2.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HomeWorld2.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HomeWorld2.php	2004-12-05 14:00:30.912000248 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/HomeWorld2.php	2004-12-05 14:02:28.731089024 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight2.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight2.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight2.php	2004-12-05 14:00:30.905001312 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight2.php	2004-12-05 14:02:28.733088720 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight.php	2004-12-05 14:00:30.905001312 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/JediKnight.php	2004-12-05 14:02:28.735088416 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/MultiTheftAuto.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/MultiTheftAuto.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/MultiTheftAuto.php	2004-12-05 14:00:30.907001008 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/MultiTheftAuto.php	2004-12-05 14:02:28.736088264 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Quake2.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Quake2.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Quake2.php	2004-12-05 14:00:30.906001160 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Quake2.php	2004-12-05 14:02:28.740087656 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/QuakeWorld.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/QuakeWorld.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/QuakeWorld.php	2004-12-05 14:00:30.911000400 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/QuakeWorld.php	2004-12-05 14:02:28.743087200 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RavenShield.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RavenShield.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RavenShield.php	2004-12-05 14:00:30.913000096 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RavenShield.php	2004-12-05 14:02:28.744087048 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RedFaction.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RedFaction.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RedFaction.php	2004-12-05 14:00:30.910000552 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/RedFaction.php	2004-12-05 14:02:28.746086744 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Savage.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Savage.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Savage.php	2004-12-05 14:00:30.907001008 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Savage.php	2004-12-05 14:02:28.748086440 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Starsiege.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Starsiege.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Starsiege.php	2004-12-05 14:00:30.908000856 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Starsiege.php	2004-12-05 14:02:28.750086136 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Tribes.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Tribes.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Tribes.php	2004-12-05 14:00:30.911000400 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Tribes.php	2004-12-05 14:02:28.751085984 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Unreal2XMP.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Unreal2XMP.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Unreal2XMP.php	2004-12-05 14:00:30.905001312 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/Unreal2XMP.php	2004-12-05 14:02:28.755085376 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament03.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament03.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament03.php	2004-12-05 14:00:30.906001160 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament03.php	2004-12-05 14:02:28.759084768 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament.php	2004-12-05 14:00:30.913000096 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery/Protocol/UnrealTournament.php	2004-12-05 14:02:28.761084464 +0100
@@ -19,7 +19,7 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Protocol.php';
+require_once 'Net/GameServerQuery/Protocol.php';
 
 
 /**
diff -burN php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery.php php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery.php
--- php-pear-Net_GameServerQuery-0.2.0.orig/Net_GameServerQuery-0.2.0/GameServerQuery.php	2004-12-05 14:00:30.903001616 +0100
+++ php-pear-Net_GameServerQuery-0.2.0/Net_GameServerQuery-0.2.0/GameServerQuery.php	2004-12-05 14:01:11.477833304 +0100
@@ -19,9 +19,9 @@
 // $Id$
 
 
-require_once 'Net\GameServerQuery\Config.php';
-require_once 'Net\GameServerQuery\Communicate.php';
-require_once 'Net\GameServerQuery\Process.php';
+require_once 'Net/GameServerQuery/Config.php';
+require_once 'Net/GameServerQuery/Communicate.php';
+require_once 'Net/GameServerQuery/Process.php';
 
 
 /**
